// processing_service.proto
// Define the processing service

syntax = "proto3";

package meteo;

import "proto/messages/meteo/meteo_messages.proto";

// ProcessingService is used to process the meteorological data
service ProcessingService {

  // Process meteorological data
  rpc ProcessMeteoData (RawMeteoData) returns (AirWellnessCoefficient);

  // Process pollution data
  rpc ProcessPollutionData (RawPollutionData) returns (PollutionCoefficient);
}

message AirWellnessCoefficient {
  float value = 1;
}

message PollutionCoefficient {
  float value = 1;
}
